app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -359,4 +382,15 @@ public void onMicrolocationsDownloadDone(MicrolocationDownloadEvent event) {
 
     }
 
+    @Subscribe
+    public void ErrorDialog(ErrorDialogEvent event){
```

Comment:
method names are improper, should be verbs and follow camel case notation. Please see the names of the functions within the file to get an idea.